pacta/build_with_tag.sh

```diff
@@ -1,51 +1,80 @@
 #! /bin/bash
 
-tag="${1:-latest}"
-clones="${2:-PACTA_analysis create_interactive_report StressTestingModelDev pacta-data}"
+# Examples:
+# build_with_tag 0.0.0.999
+
+repos="PACTA_analysis create_interactive_report StressTestingModelDev pacta-data"
+user_results="user_results"
 url="git@github.com:2DegreesInvesting/"
 
-for repo in ${clones}
-do
-    remote="${url}${repo}.git"
-    git clone -b master ${remote} --depth 1
-    echo "--"
-done
+tag="$1"
+if [ -z "$tag" ]
+then
+    echo "Please give a tag."
+    exit 0
+fi
 
-for repo in ${clones}
-do
-    git -C "${repo}" tag -a "${tag}" -m "Release pacta ${tag}" HEAD
-    echo "${repo}"
-    echo "$(git -C ${repo} log --pretty='%h %d <%an> (%cr)' | head -n 1)"
-    echo "--"
-done
+here="$(basename $(pwd))"
+if [ ! "$here" == "pacta" ]
+then
+    echo "Please run from 2diidockerrunner/pacta (not $(pwd))"
+    exit 0
+fi
```

Comment:
I believe the 2diidockerrunner/pacta contains files that are required. I'm now enforcing it as the working directory.

Reply:
yup 👍